### ISSUE TITLE
blackbox-files.txt contains full paths for files

### ISSUE THREAD

**Issue by daonin** [StackExchange/blackbox#287]
I register a new file and see in the blackbox-files.txt its full path: 
for example, 

> c/users/<user>/documents/github/notes/test.md

If I then add a new admin, re-encrypt files, commit and push and new admin will pull and try to decrypt, he will get an error of not being able to find a file by my old path, wich includes my username.

I can be doing something wrong...

**Comment by TomOnTime** [comment]
The filenames listed in blackbox-files.txt should not include the base path.  That is, they should be relative to the repo base directory.  Delete `c/users/"user"/documents/github/` from the line in`blackbox-files.txt` and it should work.

I'm not sure how that full path got in there, though I suspect that `blackbox_register_new_file` doesn't work properly under Windows. That's never been tested.

**Comment by daonin** [comment]
Yeah, I've tried that (thought it was strange to put the base path in potentially VCS and removed part of the base path in `blackbox-files.txt`) but it didn't work. I've tried different combinations (including `../`) with no luck.

**Comment by annasvst** [comment]
I have the same issue with registering files on MacOs. 
Also, when I try to decrypt a file (with blackbox_edit_start) that has been added before and are in blackbox-files.txt file I get an error that file is not found. However blackbox_decrypt_all_files works with old files.

**Comment by nboaldin** [comment]
Same issue. Blackbox referencing root of machine rather than root of repo.

**Comment by TomOnTime** [comment]
I believe this is fixed.  Please try the latest "master".

**Comment by cholantesh** [comment]
Hi; I'm experiencing the same issue with the latest version in master.
